test/setup/setup.ts

```diff
@@ -64,5 +105,5 @@ afterAll(async () => {
 	// stop only if setup did run. If using jest --watch and no tests are matched
 	// we need to not execute the require() because it will fail
 	// TODO: fix wrong typing for statsd
-	statsd.close(() => undefined);
+	// statsd.close(() => undefined);
```

Comment:
Why are we commenting this out rather than deleting? Also there is a TODO above here that should be fixed or deleted.

Reply:
Yeah, probably should delete this.